frontend/src/pages/CMT/containers/MenuChildren/components/ModelSave.js

```diff
@@ -0,0 +1,76 @@
+import React, { useContext } from "react";
+import { TbRefresh, TbCloudDownload } from "react-icons/tb";
+import { AppContext } from "../../../../../contexts/context";
+import { zipModel } from "../../../../../utils/api_misc";
+import Swal from "sweetalert2";
+import { saveAs } from "file-saver";
+
+import DropBox from "../../../../../common/components/DropBox";
+import Button from "../../../../../common/components/Button";
+
+const ModelSave = ({ refresh }) => {
+  const { drop, setDrop, parameters, setParameters } = useContext(AppContext);
+
+  const set_parameters = (e) => {
+    setDrop((prevZip) => ({
+      ...prevZip,
+      [e.target.name]: {
+        ...prevZip[e.target.name],
+        selected: e.target.value,
+      },
+    }));
+
+    setParameters({ ...parameters, [e.target.name]: e.target.value });
```

Comment:
This will result in the model chose in model zip to be used to retrain the model if user click train after choosing file in model save.